docs/pages/versions/v52.0.0/sdk/location.mdx

```diff
@@ -99,7 +100,27 @@ Learn how to configure the native projects in the [installation instructions in
 
 </ConfigReactNative>
 
-### Background Location methods
+### Background location configuration&ensp;<PlatformTag platform="ios" />
+
+To be able to run background location on iOS, you need to add the `location` value to the `UIBackgroundModes` array in your app's **Info.plist** file.
+
+**If you're using [CNG](/workflow/continuous-native-generation/)** the required `UIBackgroundModes` configuration will be applied automatically the next time you run `npx expo prebuild`.
+
+<Collapsible summary="Configure UIBackgroundModes manually on iOS">
+
+If you're not using Continuous Native Generation ([CNG](/workflow/continuous-native-generation/)) or you're using a native iOS project, then you'll need to add the following to your **Expo.plist** file:
+
+```xml ios/project-name/Supporting/Expo.plist
+<key>UIBackgroundModes</key>
+  <array>
+    <string>location</string>
+  </array>
+</key>
+```
+
+</Collapsible>
+
+### Background location methods
```

Comment:
We need all of the changes from the /docs/pages/versions/v52.0.0/sdk to also be in /docs/pages/versions/v52.0.0/unversioned

Reply:
Ok! Will update :) Thanks for pointing out.

Reply:
Guess you meant in `/docs/pages/versions/unversioned`?

Reply:
Done :)